packages/fast-jss-utilities/README.md

```diff
@@ -1,2 +1,94 @@
 # FAST - JSS Utilities
-This package (@microsoft/fast-jss-utilities) is a collection of utilities intended to be used with JSS (JavaScript Style Sheets) projects.
+This package is a collection of utilities intended to be used with JSS (JavaScript Style Sheets) projects.
+
+## Installation
+`npm i --save @microsoft/fast-jss-utilities`
+
+## Usage
+
+### Direction
+The direction is an enum which can be either ltr or rtl.
+
+```ts
+import { Direction } from "@microsoft/fast-jss-utilities";
+
+const direction = Direction.ltr;
+```
+
+### localizeSpacing
+This function should be used in conjuction with `Direction` to switch the spacing on a padding or margin.
+
+```ts
+import { Direction, localizeSpacing } from "@microsoft/fast-jss-utilities";
+
+const styles = {
+    button: {
+        padding: localizeSpacing(Direction.ltr)("8px 8px 12px 12px")
+    }
+};
+```
+
+### applyLocalizedProperty
+This function will swap the strings for the property based on the `Direction`.
+
+```ts
+import { Direction, applyLocalizedProperty } from "@microsoft/fast-jss-utilities";
+
+const styles = {
+    button: {
+        [applyLocalizedProperty("left", "right", Direction.ltr)]: "0"
+    }
+};
+```
+
+### toPx
+This method transforms a number to a string appended with `px`.
+
+```ts
+import { toPx } from "@microsoft/fast-jss-utilities";
+
+const styles = {
+    button: {
+        padding: toPx(5)
+    }
+};
+```
+
+### applyMaxLines
+The apply maxlines function is used whenever there is a limit on the number of lines shown. It takes two parameters, the number of lines and the line-height in px.
+
+```ts
+import { applyMaxLines } from "@microsoft/fast-jss-utilities";
+
+const styles = {
+    button: {
+        ...applyMaxLines(2, 24)
+    }
+};
+```
+
+### applyScreenReader
+The apply screenreader function should be used whenever there is an item that is hidden from a sighted user but visible to screen readers.

```

Comment:
Either have the method name the same (i.e. applyScreenReader instead of apply screenreader) or start the sentence like above (i.e "This method should be used whenever there is an item that is hidden from a sighted user but visible to screen readers")